source/docker/container-usage.rst

```diff
@@ -112,7 +112,10 @@ If we continue with our example, the command to launch would be the following on
 Mount custom Wazuh configuration files
 --------------------------------------
 
-To mount custom Wazuh configuration files in the Wazuh manager container, mount them in the ``/wazuh-config-mount`` folder. For example, to mount a custom ``ossec.conf`` file, mount it in ``/wazuh-config-mount/etc/ossec.conf`` and the **entrypoint.sh** script will copy the file at the right place on boot while respecting the destination file permissions.
+To use you custom Wazuh configuration files in the Wazuh manager container you have to create a folder called ``/wazuh-config-mount`` inside the container and then restart the container by using ``docker-compose restart wazuh``. 
```

Comment:
Please, review the grammar:
- `To use you` -> `to use your`.
- Use commas. The sentences are too large.
- Don't remove the example of the custom `ossec.conf` file.
- I wouldn't say that the user has to _create_ the folder inside the container, it could be understood as a kind of `mkdir`. The user should use a `volume` for it, pay attention to the previous description and try to adapt it.

Reply:
I've changed the explanations and fixed the typos, sorry for the delay